wpimath/src/main/java/edu/wpi/first/math/geometry/AllianceSymmetry.java

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.math.geometry;
+
+import java.util.HashMap;
+
+/**
+ * A utility to standardize flipping of coordinate data based on the current alliance across
+ * different years.
+ *
+ * <p>If every vendor used this, the user would be able to specify the year and no matter the year
+ * the vendor's code is from, the user would be able to flip as expected.
+ */
+public class AllianceSymmetry {
+  private AllianceSymmetry() {
+    throw new UnsupportedOperationException("This is a utility class!");
+  }
+
+  /** The strategy to use for flipping coordinates over axis of symetry. */
+  public static enum SymetryStrategy {
```

Comment:
```suggestion
  public static enum SymmetryStrategy {
```
Ctrl+f is your friend